docker-compose-dev.yml

```diff
@@ -0,0 +1,82 @@
+version: "3"
```

Comment:
This is the docker-compose file that pulls together all the projects, basically just copied from the `development` repo